src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -1194,10 +1197,28 @@
 
 (defn- application-hakukohde-review-inputs
   [review-types]
-  (into [:div.application-handling__review-hakukohde-inputs]
-        (mapv (fn [[kw label states]]
-                [application-hakukohde-review-input label kw states])
-              review-types)))
+  (let [haku-oid                          @(subscribe [:state-query [:application :selected-application-and-form :application :haku]])
+        valintalaskenta-in-hakukohteet    (->> @(subscribe [:state-query [:application :selected-review-hakukohde-oids]])
+                                               (map (fn [hakukohde-oid]
+                                                      @(subscribe [:state-query [:application :valintalaskentakoostepalvelu hakukohde-oid :valintalaskenta]]))))
+        sijoittelu?                       @(subscribe [:state-query [:haut haku-oid :sijoittelu]])
+        kevyt-valinta-feature-enabled?    (fc/feature-enabled? :kevyt-valinta)
+        display-kevyt-valinta?            (and (not sijoittelu?)
+                                               ;; false?, koska nil tarkoittaa ettei tietoa ole vielä ladattu backendiltä ja nil? palauttaisi väärän positiivisen tiedon
+                                               (every? false? valintalaskenta-in-hakukohteet))
+        show-selection-state-dropdown?    (and kevyt-valinta-feature-enabled?
```

Comment:
Nyt käy kaiketi niin, että Valinta dropdown häviää kokonaan, kunnes kevytvalinta kytketään päälle. Ajaisko
```
(or (not kevyt-valinta-feature-enabled?)
    (some true? valintalaskenta-in-hakukohteet))
```
asian?

Reply:
Ei riitä, koska `(some true? valintalaskenta-in-hakukohteet)` mahdollistaa pudotusvalikon näyttämisen kun osalla kohteista valintalaskenta ei käytössä.

Mutta korjaus

```
(or (not kevyt-valinta-feature-enabled?)
    (every? true? valintalaskenta-in-hakukohteet))
```

voisi olla paikallaan?

Reply:
Hyvältä vaikuttaa